TeachingRecordSystem/tests/TeachingRecordSystem.AuthorizeAccessToATeachingRecordTests/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+namespace TeachingRecordSystem.AuthorizeAccessToATeachingRecordTests;
+
+public class UnitTest1
```

Comment:
Did you mean to commit this?

Reply:
Yes - it prevents the workflow from blowing up complaining about no tests. It'll be gone when some real tests are added.